recipes/qt/5.x.x/conanfile.py

```diff
@@ -190,7 +192,8 @@ def config_options(self):
             self.options.opengl = "dynamic"
 
     def configure(self):
-        #if self.settings.os != "Linux":
+        if self.settings.os != "Linux":
```

Comment:
You could move it to config_options instead. The default value would be False. As settings are already evaluated, config_options seems be better. If someone pass True, validate() will fail.

Reply:
Actually, I think we should revert this change, because qt seems to be able to use gstreamer on win32 and android too (at least)